example/rbac/create_role.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+ETCD_OPERATOR_ROOT=$(dirname "${BASH_SOURCE}")/../..
+
+print_usage() {
+  echo "$(basename "$0") - Create Kubernetes RBAC role and role bindings for etcd-operator
+Usage: $(basename "$0") [options...]
+Options:
+  --role-name=STRING         Name of Role or ClusterRole to create
+                               (default=\"etcd-operator\", environment variable: ROLE_NAME)
+  --role-binding-name=STRING Name of RoleBinding or ClusterRoleBinding to create
+                               (default=\"etcd-operator\", environment variable: ROLE_BINDING_NAME)
+  --namespace=STRING         namespace to create role and role binding in. Must already exist.
+                               (default=\"default\", environment vairable: NAMESPACE)
+" >&2
+}
+
+ROLE_NAME="${ROLE_NAME:-etcd-operator}"
+ROLE_BINDING_NAME="${ROLE_BINDING_NAME:-etcd-operator}"
+NAMESPACE="${NAMESPACE:-default}"
+
+for i in "$@"
+do
+case $i in
+    --role-name=*)
+    ROLE_NAME="${i#*=}"
+    ;;
+    --role-binding-name=*)
+    ROLE_BINDING_NAME="${i#*=}"
+    ;;
+    --namespace=*)
+    NAMESPACE="${i#*=}"
+    ;;
+    -h|--help)
+      print_usage
+      exit 0
+    ;;
+    *)
+      print_usage
+      exit 1
+    ;;
+esac
+done
+
+version_yaml=$(kubectl version --client -o yaml)
```

Comment:
- This doesn't work for 1.7 kubectl.
- Can you remove this part of k8s version check? We already mention in "Requirements" section on k8s version. We don't need to duplicate this logic in every script which uses kubectl. Thanks!

Reply:
Removed